Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Material/Material.h

```diff
@@ -165,9 +173,18 @@ namespace AZ
             bool TryApplyPropertyConnectionToShaderEnable(
                 const MaterialPropertyValue & value,
                 const MaterialPropertyOutputId & connection);
```

Comment:
Note to reviewers: the above TryApply functions are new too, but I forgot to remove the declarations before committing an earlier PR. The definitions in cpp are new in this PR.